xml/ha_fencing.xml

```diff
@@ -362,11 +362,13 @@
     </para>
 <screen>&prompt.root;<command>crm configure</command>
 &prompt.crm.conf;<command>primitive st-ibmrsa-1 stonith:external/ibmrsa-telnet \
-params nodename=&node1; ip_address=192.168.0.101 \
-username=USERNAME password=PASSW0RD</command>
+  params nodename=&node1; ip_address=192.168.0.101 \
+  username=USERNAME password=PASSW0RD \
+  op monitor timeout=20s interval=3600s</command>
```

Comment:
This changes are not needed. There was the long debate in the past. In general, we don't add "monitor" for all fencing devices in CIB.